src/ResourceManagement/AnalysisServices/AnalysisServices.Tests/Helpers/AnalysisServicesTestUtilities.cs

```diff
@@ -69,7 +69,7 @@ public static class AnalysisServicesTestUtilities
             "aztest1@stabletest.ccsctp.net"
         };
 
-        public static string GetDefaultCreatedResponse(string provisioningState)
+        public static string GetDefaultCreatedResponse(string state)
         {
```

Comment:
Take both parameters provisioningstate and state. Change the caller accordingly.